presto-docs/src/main/sphinx/release/release-335.rst

```diff
@@ -0,0 +1,51 @@
+=========================
+Release 335 (XX Jun 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix failure when :func:`reduce_agg` is used as a window function. (:issue:`3883`)
+* Fix incorrect cast from ``TIMESTAMP WITHOUT TIME ZONE`` to ``TIME`` type. (:issue:`3848`)
+* Fix incorrect query results when converting very large ``TIMESTAMP`` values into
+  ``TIMESTAMP WITHOUT TIME ZONE``, or when parsing very large
+  ``TIMESTAMP WITHOUT TIME ZONE`` values. (:issue:`3956`)
+* Improve optimized local scheduling with regard to non-uniform data distribution. (:issue:`3922`)
+* Add support for variable-precision ``TIMESTAMP WITHOUT TIME ZONE`` type. (:issue:`3783`)
+* Return ``VARCHAR`` type when :func:`substr` argument is ``CHAR`` type. (:issue:`3599`)
```

Comment:
This also fixes the returned value.

Also link to https://github.com/prestosql/presto/issues/3456 (instead? additionally?)

Reply:
Probably in addition..

Reply:
Move this above with the fixes, since this is a fix/change to existing behavior.